commitlint.config.js

```diff
@@ -1,5 +1,8 @@
 const config = {
   extends: ['@commitlint/config-conventional'],
+  rules: {
+    'subject-case': [0, 'always', 'upper-case'],
```

Comment:
Needed to allow upper case in the subject case. I need to pass the JIRA ticket to the commit message and the ticket number are all in capital.